src/components/field/index.tsx

```diff
@@ -294,17 +296,18 @@ export interface DraggedFieldIdentifier {
   fieldDef: ShelfFieldDef;
   parentId: FieldParentId;
   filter: RangeFilter | OneOfFilter;
+  filters: Array<RangeFilter | OneOfFilter>;
```

Comment:
Why do we need the whole filters array when we drag only one thing? This seems pretty coupled?